### ISSUE TITLE
`createCommitOnBranch` is null from GitHub GraphQL API

### ISSUE THREAD

**Issue by waynevanson** [sveltia/sveltia-cms#130]
When I click save, I'm met with "createCommitOnBranch is null".

This means the `commit` property on the graphql response for `createCommitOnBranch` is null: `createCommitOnBranch.commit === null`

- Repository
  https://github.com/waynevanson/waynevanson/tree/cba2034871dcccc1f3c5cc3c7532fa575ff8c719

If there's more information I can provide, please let me know :)

Here are the GraphQL API's

<details>
<summary>graphql.repository request (unminified)</summary>

```json
{
  "query": " query { repository(owner: \"waynevanson\", name: \"waynevanson\") { ref(qualifiedName: \"main\") { target { oid } } } } ",
  "variables": {}
}
```
</details>

<details>
<summary>graphql.repository response</summary>

```json
{
    "data": {
        "repository": {
            "ref": {
                "target": {
                    "oid": "cba2034871dcccc1f3c5cc3c7532fa575ff8c719"
                }
            }
        }
    }
}
```
</details>

The second graphql call returns an error.

<details>
<summary>graphql.createCommitOnBranch request (unminified)</summary>

```json
{
  "query": " mutation ($input: CreateCommitOnBranchInput!) { createCommitOnBranch(input: $input) { commit { url } } } ",
  "variables": {
    "input": {
      "branch": {
        "repositoryNameWithOwner": "waynevanson/waynevanson",
        "branchName": "main"
      },
      "expectedHeadOid": "cba2034871dcccc1f3c5cc3c7532fa575ff8c719",
      "fileChanges": {
        "additions": [
          {
            "path": "/src/content/posts/title-of-post.md",
            "contents": "LS0tCnRpdGxlOiBUaXRsZSBvZiBwb3N0CnN1bW1hcnk6IFN1bW1hcnkgb2YgcG9zdAotLS0KQm9keSBvZiBwb3N0Cg=="
          }
        ],
        "deletions": []
      },
      "message": { "headline": "Create Posts “title-of-post”" }
    }
  }
}
```

</details>

<details open>
<summary>graphql.createCommitOnBranch response (error)</summary>

```json
{
    "data": {
        "createCommitOnBranch": null
    },
    "errors": [
        {
            "type": "NOT_FOUND",
            "path": [
                "createCommitOnBranch"
            ],
            "locations": [
                {
                    "line": 1,
                    "column": 50
                }
            ],
            "message": "A path was requested for deletion which does not exist as of commit oid `cba2034871dcccc1f3c5cc3c7532fa575ff8c719`"
        }
    ]
}
```

</details>

Other sveltia related files.

<details>
<summary>admin/config.yml</summary>

```yaml
backend:
  name: github
  branch: main
  repo: waynevanson/waynevanson
media_folder: /src/media
public_folder: /public
collections:
  - name: posts
    label: Posts
    folder: /src/content/posts
    create: true
    fields:
      - label: Title
        name: title
        widget: string
      - label: Summary
        name: summary
        widget: string
      - label: Body
        name: body
        widget: markdown
```
</details>

<details>
<summary>example collection item</summary>

```md
---
title: Hello
summary: World
---
## I'm mark down, aren't I! 

How exciting!

```

</details>

<details>
<summary>admin/index.html</summary>

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta name="robots" content="noindex" />
    <title>Content Manager</title>
    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
  </head>
  <body>
    <!-- Include the script that builds the page and powers Sveltia CMS -->
    <script
      src="https://unpkg.com/@sveltia/cms/dist/sveltia-cms.js"
      type="module"
    ></script>
  </body>
</html>
```
</details>

**Comment by kyoshino** [comment]
Thanks for the bug report, @waynevanson! I’ll take a look at what’s going on.

**Comment by kyoshino** [comment]
Found the cause. The [`folder`](https://github.com/waynevanson/waynevanson/blob/cba2034871dcccc1f3c5cc3c7532fa575ff8c719/public/admin/config.yml#L10) path unexpectedly starts with `/`. Removing the trailing slash should solve the issue.

```diff
-folder: /src/content/posts
+folder: src/content/posts
```

I’ll update Sveltia CMS to remove the `/` anyway.

**Comment by kyoshino** [comment]
Just shipped [v0.20.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.20.0) to work around the issue.

**Comment by waynevanson** [comment]
Nice @kyoshino! DecapCMS handle this correctly, so the feature parity is nice.

Thanks for the help!
